src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -93,7 +88,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </Target>
 
     <Target Name="CppWinRTCalculateEnabledProjections"
-            DependsOnTargets="GetCppWinRTProjectWinMDReferences;GetCppWinRTMdMergeInputs"
+            DependsOnTargets="GetCppWinRTProjectWinMDReferences;GetCppWinRTMdMergeInputs;$(CppWinRTCalculateEnabledProjectionsDependsOn)"
```

Comment:
left the DependsOn prop.  If not intentional, let's remove here and elsewhere.  If intentional (somebody needs a hook), let's add to all targets for consistency (we do that later, it's not related to this PR).  

Again, we shouldn't implement our own dependencies in turn on public DependsOn props, as users could easily break that if they forget the pattern of appending to existing DependsOn values.

Reply:
I left this on intentionally, as it seems like a useful place for extension.

I don't think there any any dependencies implemented with our public DependsOn props anymore?